docker-compose.yaml

```diff
@@ -27,27 +27,28 @@ services:
     container_name: zomboid-dedicated-server
     restart: "no"
     environment:
-      - "ADMIN_PASSWORD=changeme"
-      - "ADMIN_USERNAME=superuser"
-      - "AUTOSAVE_INTERVAL=15"
-      - "BIND_IP=0.0.0.0"
-      - "DEFAULT_PORT=16261"
-      - "GAME_VERSION=public"
-      - "GC_CONFIG=ZGC"
-      - "MAP_NAMES=Muldraugh, KY"
-      - "MAX_PLAYERS=16"
-      - "MAX_RAM=4096m"
-      - "MOD_NAMES="
-      - "MOD_WORKSHOP_IDS="
-      - "PAUSE_ON_EMPTY=true"
-      - "PUBLIC_SERVER=true"
-      - "RCON_PASSWORD=changeme_rcon"
-      - "RCON_PORT=27015"
-      - "SERVER_NAME=ZomboidServer"
-      - "SERVER_PASSWORD="
-      - "STEAM_VAC=true"
-      - "UDP_PORT=16262"
-      - "USE_STEAM=true"
+      ADMIN_PASSWORD: "changeme"
+      ADMIN_USERNAME: "superuser"
+      AUTOSAVE_INTERVAL: "15"
+      BIND_IP: "0.0.0.0"
+      DEFAULT_PORT: "16261"
+      GAME_VERSION: "public"
+      GC_CONFIG: "ZGC"
+      MAP_NAMES: "Muldraugh, KY"
+      MAX_PLAYERS: "16"
+      MAX_RAM: "4096m"
+      MOD_NAMES: ""
+      MOD_WORKSHOP_IDS: ""
+      PAUSE_ON_EMPTY: "true"
+      PUBLIC_SERVER: "true"
+      RCON_PASSWORD: "changeme_rcon"
+      RCON_PORT: "27015"
+      SERVER_NAME: "ZomboidServer"
+      SERVER_PASSWORD: ""
+      STEAM_VAC: "true"
+      UDP_PORT: "16262"
+      USE_STEAM: "true"
+      TZ: "UTC"
```

Comment:
Not sure why you changed this from array to map format, but it was still valid before according to the [docs](https://docs.docker.com/compose/compose-file/#environment).

If you really want to change this, you can, but you will need to update the docker-compose test so that the sed command works again: [.github/workflows/docker-build.yml](https://github.com/Renegade-Master/zomboid-dedicated-server/blob/0edfde925f1c98f88a1fe009d66e9b75ab509e38/.github/workflows/docker-build.yml#L108)

Reply:
There's no particular reason, I just prefer this way.
I've rolled back to how it was and just added the TZ env